assets/styles/default-theme.scss

```
@@ -282,6 +282,13 @@ input[type="submit"]:hover, button[type=submit]:hover {background-color: var(--r
     margin-bottom: 1rem;
 }
 
+.rb-view-repeatable-table th,
+.rb-view-repeatable-table td,
+.rb-view-repeatable-table td a {
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)